### ISSUE TITLE
Accessibility Review: Visibility Classes

### ISSUE THREAD

**Issue by gakimball** [zurb/foundation-sites-6#2]
Paging @gerardkcohen.

**Files:**
- [CSS for visibility classes](https://github.com/zurb/foundation-sites-6/blob/master/scss/components/_visibility.scss)
- [CSS for making invisible elements](https://github.com/zurb/foundation-sites-6/blob/master/scss/util/_mixins.scss#L113)

**Features:**
- `.hide` and `.invisible` classes which add `display: none` and `visibility: hidden`.
- Responsive `.show-for-*` and `.hide-for-*` classes which toggle `display` between `block` and `none`.
- `.show-for-sr` class which visually hides an element while still allowing AT to read it.
- `.show-on-focus` class which mimics `.show-for-sr`, but reveals the element when focused.

**Questions:**
- How do screen readers report their viewport? Is it something we should be concerned about, or should we just make a note of it in the documentation?
- Do the hidden states of our visibility classes need `visibility: hidden` as well? I seem to recall this being a bug in some screen readers.
